test/csv_fast_importer_test.rb

```diff
@@ -115,6 +115,23 @@
     end
   end
 
+  describe 'with "-" in destination table name' do
+
+    before do
+      ActiveRecord::Base.connection.execute <<-SQL
+        DROP TABLE IF EXISTS "special-character";
+        CREATE TABLE "special-character" ( label varchar NULL );
+      SQL
+      csv_writer = CSVWriter.new 'special-character.csv'
```

Comment:
Maybe. Is it a problem?
If you can call `CsvFactImporter` (with SQL injection purpose), you also have access to `ActiveRecord::Base.connection` which allow you to call any SQL query directly.
I mean, `CsvFactImporter` is a backend api, not designed to handle external parameters.